pkg/config/runtime.go

```diff
@@ -151,8 +152,16 @@ func resolve(bag attribute.Bag, kindSet KindSet, rules map[rulesKey]*pb.ServiceC
 			glog.Warningf("%s attribute not found in %p", identityAttribute, bag)
 			return nil, fmt.Errorf("%s attribute not found", identityAttribute)
 		}
-	} else if scopes, err = GetScopes(attr.(string), identityAttributeDomain, scopes); err != nil {
-		return nil, err
+	} else {
+		attrStr, ok := attr.(string)
+		if !ok {
+			msg := fmt.Sprintf("%s attribute should be of type string", identityAttribute)
```

Comment:
Message should include "%v",attr so operator knows what was sent

Reply:
Done